### DISCUSSION TITLE
How do I know nvim-treesitter is working for this buffer?

### DISCUSSION THREAD

**Discussion by Lilja** [nvim-treesitter/nvim-treesitter#3247] [Q&A]
I've spent about an hour trying to debug why the vue formatting wasn't working very well. I disabled everything only to find everything the same, meaning I've been using neovim's syntax highlighting.

Is there a way to get logs from this plugin? How do I know it's working? Is there a way to check?

**Comment by kyazdani42** [comment]
use ts playground https://github.com/nvim-treesitter/playground

**Reply by kyazdani42** [reply]
or `:checkhealth nvim-treesitter` also.

**Reply by Lilja** [reply]
I tried to use the playground, unfortunately it doesn't work to check what's wrong.

Here is `:TSPlaygroundToggle` with 
```lua
require'nvim-treesitter'.setup {
    highlight { enable = false }
}
```
<details>

<summary>Playground showing information when enabled is explicitly is set as disabled</summary>

<img width="1280" alt="bild" src="https://user-images.githubusercontent.com/6134511/182137215-9478b6f5-3814-41f0-812e-2b30a809590b.png">

</details>


<details>

<summary>`:TSCaptureUnderCursor` showing posva/vim-vue syntax highlighting</summary>

<img width="1280" alt="bild" src="https://user-images.githubusercontent.com/6134511/182137338-17b5e0dd-64a6-44d9-9e3a-c8f2ab0bf403.png">

</details>

<details>

<summary>Enabled, look how `defineProps` has changed color.</summary>

<img width="1279" alt="bild" src="https://user-images.githubusercontent.com/6134511/182137763-efa86ca6-25d3-421d-a85e-c3e76a3f62c7.png">

</details>

<details>

<summary>Same image but capture under cursor</summary>

<img width="1280" alt="bild" src="https://user-images.githubusercontent.com/6134511/182137852-900aad4b-d173-4d81-9fc0-68e010271f88.png">

</details>

**Reply by kyazdani42** [reply]
seems like TSHighlightCapturesUnderCursor shows the information on whether this is treesitter or vim syntax right ?
